dist/index.html

```diff
@@ -1 +1,56 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Document</title><script defer="defer" src="bundle.js"></script></head><body><h1 class="prueba">prueba</h1></body></html>
\ No newline at end of file
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Salsa&display=swap" rel="stylesheet">
+<script defer src="bundle.js"></script></head>
+<body>
+    <main class="main-section">
+      <div class="main-section-title">
+        <h1 class="main-title-text">Leaderboard</h1>
+      </div>
+      <div class="main-app">
+        <div class="leaderboard-container">
+          <div class="leaderboard-title-container">
+            <h3 class="recent-scores-title">Recent scores</h3>
+          <button class="refresh-btn">Refresh</button>
+          
+          </div>
+          <div class="leaderboard">
+            <div class="score">Name:100</div>
+            <div class="score">Name:20</div>
+            <div class="score">Name:50</div>
+            <div class="score">Name:78</div>
+            <div class="score">Name:125</div>
+            <div class="score">Name:77</div>
+            <div class="score">Name:42</div>
+          </div>
+        </div>
+        <div class="form-container">
+          <h3 class="form-title">Add your score</h3>
+          <form class="form">
+            <input 
+            class="name-input"
+            requiered
+            type="text"
+            placeholder="Your name">
+            <input 
+            class="score-input"
+            requiered
+            type="text"
+            placeholder="Your score">
```

Comment:
- [Optional] Dear @maherramadan78, Good job on the project. Please consider making your `score` type a `number` instead of text. This is to make sure only a number can be submitted as a value.